packages/utilities/gel-foundations/scripts.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable global-require */
+module.exports = {
+  news: require('./dist/scripts/news').default,
```

Comment:
```suggestion
```
News should not be a separate script. We should be using `latin`. 

Currently, the `news` file has these extra values that are missing in the `latin` file:
`atlas`, `elephant`, `imperial`, `royal`, `foolscap`, `paragon`, `doublePica`, `greatPrimer`, `bodyCopy`. 
Can you please add them?

I saw your comment below ```There is a distinction between latin.js and news.js because we assumed it would make it easier for UX to change something in WS and not in NEWS, and at the moment we don't have Body Copy in WS and we are using its sizes as Pica.```

However, News UX and WS UX has to be unified and not independent, otherwise our shared approach will not work. News is just a special case of a Latin WS site.